docker-compose.yml

```diff
@@ -43,6 +46,8 @@ services:
       traefik.http.routers.frontend.tls: true
 
   jsonrpc:
+    image: yeagerai/simulator-jsonrpc:${STUDIO_DOCKER_IMAGE_VERSION:-latest}
+    pull_policy: always
```

Comment:
are you sure this doesn't interfere with local development?

I'm pretty sure from past experiences that this breaks local development